airflow/api_connexion/openapi/v1.yaml

```diff
@@ -19,7 +19,29 @@ openapi: 3.0.3
 
 info:
   title: "Airflow API (Stable)"
-  description: Apache Airflow management API.
+  description: |
+    Apache Airflow management API.
+
+    # Errors
+
+    We follow the error response format proposed in [RFC 7807](https://tools.ietf.org/html/rfc7807)
+    also known as Problem Details for HTTP APIs.  As with our normal API responses,
+    your client must be prepared to gracefully handle additional members of the response.
+    ## Unauthenticated
+    <RedocResponse pointer={"#/components/responses/Unauthenticated"} />
```

Comment:
This element is not supported in the Open Source Redoc version. I would just write 2-3 sentences here what this error message means and when it happens and what can be done to fix it.